userManagement/templates/userManagement/userRegister.html

```diff
@@ -0,0 +1,7 @@
+{% extends "itemManagement/chrome_template.html" %}
```

Comment:
I think we should start commenting descriptions of html pages and what they do. It would be especially helpful in this case where this page has no functionality at all.